src/box/allocator.h

```diff
@@ -0,0 +1,153 @@
+#ifndef TARANTOOL_BOX_ALLOCATOR_H_INCLUDED
+#define TARANTOOL_BOX_ALLOCATOR_H_INCLUDED
```

Comment:
Please, use `#pragma once`. This is a new rule, which is not added anywhere yet, sorry. Kirill approved it and was supposed to update the documentation.

Reply:
Fixed